core/cautils/fileutils_test.go

```diff
@@ -30,8 +37,16 @@ func TestListFiles(t *testing.T) {
 	assert.Equal(t, 12, len(files))
 }
 
+func TestLoadResourcesFromFilesEmptyFiles(t *testing.T) {
```

Comment:
`LoadResourcesFromFilesEmptyFiles` is not a function. This is an specific test case for the  `LoadResourcesFromFiles` function. As such, should be under `TestLoadResourcesFromFiles`